app/services/report_importer.rb

```diff
@@ -49,7 +52,7 @@ def scan
     report_scanners.each do |scanner|
       break if scanner.scan(report, logs)
     end
-    logger.debug "Changes after scanning: #{report.changes.inspect}"
+    logger.debug { "Changes after scanning: #{report.changes.inspect}" }
```

Comment:
Did you use the proc call to delay evaluation of the content, so that  `report.changes` can be changed prior to the time the `log.debug` is evaluated?

Reply:
Yes, importers are under heavy load and every line count. We use this on other lines in this file already, I was thinking to do the same.